### PR TITLE
HMS-4102/4104: Adding gif and video support as background for VB

### PR DESCRIPTION
**Pull request by A-KVerma**

### CODE REVIEW

File: docs/javascript/v2/plugins/virtual-background.mdx [hunk 1]
```diff
@@ -42,6 +42,7 @@ This accepts background as a parameter, It can be of 3 types:
 -  `blur` - String - This will set the background to blur
 -  `image` - HTMLImageElement - This will replace the background to the image provide
 -  `none` - String - This will remove the background effect from the video
+-  `gifURL` - String - This is the URL of the gif with which the background will be replaces
```
Comment (triptu) [inline docs/javascript/v2/plugins/virtual-background.mdx:45]: replaces => replaced

Video also needs to be added right?

File: docs/javascript/v2/plugins/virtual-background.mdx [hunk 2]
```diff
@@ -149,14 +150,14 @@ function VirtualBackground() {
 ### Change Background
 
 The function `setBackground` can be used to update the background again later.
-It accepts string or HTMLImageElement as a parameter
-Image will get fit to the video by maintaining the aspect ratio.
-If the aspect ratio of the background image is not the same as the video, the image will be cropped to fit in the background.
+It accepts (string or HTMLImageElement or HTMLVideoElement) as a parameter
```
Comment (triptu) [inline docs/javascript/v2/plugins/virtual-background.mdx:153]: it looks better without the brackets

File: docs/javascript/v2/plugins/virtual-background.mdx (hunk 1 above)
Comment (A-KVerma) [reply docs/javascript/v2/plugins/virtual-background.mdx:45]: done

File: docs/javascript/v2/plugins/virtual-background.mdx [hunk 3]
```diff
@@ -40,12 +40,14 @@ import HMSVirtualBackgroundPlugin from '@100mslive/hms-virtual-background';
 This accepts background as a parameter, It can be of 3 types:
 
 -  `blur` - String - This will set the background to blur
--  `image` - HTMLImageElement - This will replace the background to the image provide
+-  `image` - HTMLImageElement - This will replace the background to the image provided
+-  'video' - HTMLVideoElement - This will replace the background to the video provided
```
Comment (triptu) [inline docs/javascript/v2/plugins/virtual-background.mdx:44]: backticks instead of single quote around video

**Review by triptu — APPROVED** [review]

**Pull request closed without merging**